backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/navigationbar/internal/BpkTopNavBarImpl.kt

```diff
@@ -60,10 +61,12 @@ internal fun BpkTopNavBarImpl(
     insets: WindowInsets? = null,
     navIcon: IconAction? = null,
     actions: List<Action> = emptyList(),
+    transparent: Boolean = false,
 ) {
+    val backgroudColor = if (fraction == 0f) BpkTheme.colors.surfaceDefault else (if (transparent) Color.Transparent else BpkTheme.colors.canvas)
```

Comment:
this is a little hard to read. can you add {} to the if/else and reformat to multiple lines

Reply:
also small typo - `backgroundColor`

Reply:
ok, will update it, thanks.